doc/changes/unreleased.md

```diff
@@ -8,4 +8,5 @@
 * [#372](https://github.com/exasol/python-toolbox/issues/372): Added conversion from pip-audit JSON to expected GitHub Issue format
```

Comment:
Should add summary of changes at top with any additional usage changes, particularly for different poetry versions
Breaking change